packages/cpt-ui/src/context/PrescriptionInformationProvider.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {
+  createContext,
+  useContext,
+  useState,
+  useEffect,
+  ReactNode
+} from "react"
+import {useLocation} from "react-router-dom"
+
+import {normalizePath} from "@/helpers/utils"
+import {FRONTEND_PATHS} from "@/constants/environment"
+
+export interface PrescriptionInformation {
```

Comment:
If this data is being fetched from the backend and stored directly, it's better to have a type defined in the `common-types` package, and use that. If there's any parsing or extraction that needs to be done, that should be done by the banner.